### PR TITLE
Refactor the figures type

### PR DESCRIPTION
**Pull request by bassjobsen**
Refactoring according the comments on https://github.com/twbs/bootstrap/pull/18104.

### CODE REVIEW

**Comment by mdo** [comment]
The figure CSS has been moved to images recently, so this will need updating <3.

**Comment by cvrebert** [comment]
@mdo Other than that, does this look okay? I can do the updating when merging this.

File: docs/content/figures.md [hunk 1]
```diff
@@ -6,11 +6,11 @@ group: content
 
 Anytime you need to display a piece of content—like an image—with an optional caption, consider using a `<figure>`.
 
-Use the included `.figure` and `.figure-caption` classes to provide some baseline styles for the HTML5 `<figure>` and `<figcaption>` elements. As a bonus, immediate children images are automatically responsive.
+Use the included `.figure` , `.figure-img` and `.figure-caption` classes to provide some baseline styles for the HTML5 `<figure>` and `<figcaption>` elements. You will have to add the `.img-fluid` class to your `img` to make it responsive.
```
Comment (mdo) [inline docs/content/figures.md]: I'd clarify that new line at the end with something like:

> Images in figures have no explicit size, so be sure to add `.img-fluid` to your `` to make it responsive.

**Comment by mdo** [comment]
Other than that one comment, go for it.

**Comment by cvrebert** [comment]
Merged as 0d34169e1f1126c0e752694905c037278402e0c8.
Thanks @bassjobsen!

**Pull request closed without merging**